source/_integrations/greencell.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: Greencell
+description: Instructions on how to use the Greencell EVSE integration in Home Assistant.
+ha_category:
+  - Button
+  - Number
+  - Sensor
+ha_release: 2025.5.1
+ha_codeowners:
+  - '@BrzezowskiGC'
+ha_domain: greencell
+ha_integration_type: integration
+---
+
+The Greencell EVSE [HabuDen](https://greencell.global/en/555-ev-chargers#/power-22kw_8_stage_regulation) integration for Home Assistant enables:
```

Comment:
In the first line, the integration is called Greencell EVSE HabuDen. In the title it is called Greencell. What is the actual name of the integration?